### ISSUE TITLE
Filter by several tags

### ISSUE THREAD

**Issue by thorstengrothe** [Strauman/exercisebank#112]
I asked this somewhere in the past. But I guess it was the wrong location.

Say we have this construct

```
\nextproblem{points=2,tag={a,b}}
\begin{problem}
    Tag Test
\end{problem}
```
Now I want to filter with this command. 

```
\buildtags{%
     % a, 
    b
}{File}
```
I think it would be more logical if only `a` **and** `b` would match and not `a` **or** `b`.
Is this possible?

Thorsten

**Comment by Strauman** [comment]
Good point. I guess it should be written explicit somewhere. e.g. `\buildtagsAnd` and `\buildtagsOr`.
